walkeeper/src/json_ctrl.rs

```diff
@@ -0,0 +1,218 @@
+//!
+//! This module implements JSON_CTRL protocol, which allows exchange
+//! JSON messages over psql for testing purposes.
+//!
+//! Currently supports AppendLogicalMessage, which is used for WAL
+//! modifications in tests.
+//!
+
+use anyhow::{anyhow, Result};
+use bytes::{BufMut, Bytes, BytesMut};
+use crc32c::*;
```

Comment:
looks like we use only crc32c_append from this package, so it can be imported directly without wildcard use